openwisp_radius/api/serializers.py

```diff
@@ -102,7 +103,20 @@ class Meta:
         fields = '__all__'
 
 
+class RadiusOrganizationField(serializers.SlugRelatedField):
+    def get_queryset(self):
+        queryset = Organization.objects.all()
+        request = self.context.get('request', None)
+        if not request.user.is_superuser:
+            queryset = queryset.filter(pk__in=request.user.organizations_dict)
```

Comment:
shouldn't this be `request.user.organizations_dict.keys()`?

Reply:
It works either way, filter only looks at the keys. :smile:
I'll add `keys()` to help code be more readable / obvious in what it is doing! :smile: